bin/pry

```diff
@@ -17,6 +19,10 @@ def udec(*)
   UBID.decode(*)
 end
 
+def udec_deleted(ubid)
+  DeletedRecord.where(Sequel.pg_json(:model_values)["name"] => Sequel.pg_jsonb_wrap(UBID.parse(ubid).to_uuid)).first
+end
```

Comment:
What do you think about adding it to the `udec` as a second option?
Probably we will run `udec` first and it will return `nil`, then we will run `udec_deleted`
```suggestion
  UBID.decode(*) || DeletedRecord.where(Sequel.pg_json(:model_values)["name"] => Sequel.pg_jsonb_wrap(UBID.parse(ubid).to_uuid)).first
end
```